### DISCUSSION TITLE
Installation for NixOS Users?

### DISCUSSION THREAD

**Discussion by cgumb** [siduck/chadwm#176] [General]
Howdy! 👋
I was curious if anyone has tried installing `chadwm` with nix/NixOS.

Specifically, I'm not sure how to handle the additional steps required by the installation: moving the eww directory, changing into the chadwm directory, etc.

Trying to simply replicate the commands shown in the repo's installation instructions inside the buildPhase of the nix configuration did not work (it didn't seem to see those directories).

I know siduck's `st` build repo included a default.nix and flake.nix file.
Perhaps another nix user would be willing to share the steps they took to get chadwm installed?

Thanks! 🤓

**Comment by erikdubois** [comment]
can we ask your help to get Chadwm up and running on NixOS.

I tried, failed and asked the Nixos developer to create a pkgbuild.

https://github.com/NixOS/nixpkgs/issues/282239

can you assist in anyway?

failure to build is here : https://github.com/NixOS/nixpkgs/issues/281587
